Alley-Interactive/Sniffs/PHP/StrictTypeDeclarationSpacingSniff.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * StrictTypeDeclarationSpacingSniff class file
+ *
+ * @package Alley\CodingStandards
+ */
+
+namespace Alley\Sniffs\PHP;
+
+use WordPressCS\WordPress\Sniff;
+
+class StrictTypeDeclarationSpacingSniff extends Sniff {
+	/**
+	 * Returns tokens to listen for.
+	 *
+	 * @return array<int|string>
+	 */
+	public function register() {
+		return [ T_DECLARE ];
+	}
+
+	/**
+	 * Processes this sniff when one of its tokens is encountered.
+	 *
+	 * @param int $stackPtr The position of the current token in the stack.
+	 */
+	public function process_token( $stackPtr ) {
+		$tokens = $this->phpcsFile->getTokens();
+
+		// Find strict_types declaration.
+		$strictTypePtr = $this->phpcsFile->findNext(T_STRING, $stackPtr + 1, null, false, 'strict_types');
+		if ( $strictTypePtr === false ) {
+			return;
+		}
+
+		// Check for the opening parenthesis.
+		$openParenPtr = $this->phpcsFile->findNext(T_OPEN_PARENTHESIS, $stackPtr + 1);
+		if ($openParenPtr !== false) {
+
+			// Check for space after OPENING parenthesis.
+			$nextTokenPtr = $openParenPtr + 1;
+			if ($tokens[$nextTokenPtr]['code'] === T_WHITESPACE) {
+				$error = 'No space allowed after opening parenthesis in strict_types declaration';
+				$fix   = $this->phpcsFile->addFixableError($error, $nextTokenPtr, 'SpaceAfterOpenParenthesis');
+				if ($fix) {
+					$this->phpcsFile->fixer->replaceToken($nextTokenPtr, '');
```

Comment:
I'll add unit tests for the phpcbf part in another pr since I need to setup our unit tests to support it.